treeherder/push_health/tests.py

```diff
@@ -123,6 +123,16 @@ def get_current_test_failures(push, option_map, jobs):
         isClassifiedIntermittent = any(
             job['failure_classification_id'] == 4 for job in jobs[job_name]
         )
+        is_investigated = any(
```

Comment:
I would think this would work and be quick.  Is this what is too slow?  I wonder if you first group all `investigated_tests` by the `job_type`?  Then you could check any test to see if it's in that `dict` and check inside it if it has a matching `test_name`?

Reply:
No, this is not at all slow. 
I was speaking of things in frontend
When we check against any test, it goes on to add that test in the state variable in selectedTests (in the Test.jsx file)
I see that the check is not done instantly but took a bit of time on my PC

> Another approach would be do download the investigated tests for the push and match them up in the UI/JavaScript

Should I try doing it like this. This would need another API call and I see no big advantage of this too

ALso, for making the things real time ( disabling the tests as soon as it marked investigated) , should I make anothet get request to the push health page or manually add things to the state.

Reply:
@SuyashSalampuria : I would not re-fetch the data.  I would update the test on the front-end when you get a "success" response from the server for taking it investigated.  We re-fetch the data every two minutes (iirc) which will catch if another person (or same person in another page) updated something.

Reply:
slight nit; isClassified vs is_investigated, we should keep camelCase